staging/cse/windows/provisioningscripts/cleanupnetwork.ps1

```diff
@@ -30,8 +34,27 @@ if ($hnsNetwork) {
     Get-HnsPolicyList | Remove-HnsPolicyList
 
     Write-Host "Cleaning up old HNS network found"
-    Remove-HnsNetwork $hnsNetwork
-    Start-Sleep 10
+
+    if (-not $IsCleanupHnsNetwork) {
+        # Original code
+        Remove-HnsNetwork $hnsNetwork
+        Start-Sleep 10
+    } else {
```

Comment:
should always remove hnsendpoint first and then hnsnetwork.. we can keep that part common